libs/prisma-models/src/sql_ext/column.rs

```diff
@@ -153,11 +155,33 @@ where
         let table = sf.model().db_name().to_string();
         let col = sf.db_name().to_string();
 
-        let column = Column::from(((db, table), col));
+        let type_family = quaint::ast::TypeFamily::from(&sf.type_identifier);
+        let column = Column::from(((db, table), col)).type_family(type_family);
 
         match sf.default_value.as_ref().and_then(|d| d.get()) {
             Some(default) => column.default(sf.value(default)),
             None => column.default(quaint::ast::DefaultValue::Generated),
         }
     }
 }
+
+impl From<&TypeIdentifier> for quaint::ast::TypeFamily {
+    fn from(ti: &TypeIdentifier) -> Self {
+        use quaint::ast::TypeFamily;
+
+        match ti {
+            TypeIdentifier::String => TypeFamily::Text,
+            TypeIdentifier::Int => TypeFamily::Int,
+            TypeIdentifier::BigInt => TypeFamily::Int,
+            TypeIdentifier::Float => TypeFamily::Double,
+            TypeIdentifier::Decimal => TypeFamily::Decimal,
+            TypeIdentifier::Boolean => TypeFamily::Boolean,
+            TypeIdentifier::Enum(_) => TypeFamily::Text,
+            TypeIdentifier::UUID => TypeFamily::Uuid,
+            TypeIdentifier::Json => TypeFamily::Text,
+            TypeIdentifier::Xml => TypeFamily::Text,
+            TypeIdentifier::DateTime => TypeFamily::DateTime,
+            TypeIdentifier::Bytes => TypeFamily::Bytes,
+        }
+    }
+}
```

Comment:
Can we try to solve this in a different way so this is part of the `sql-query-connector` instead? As @yoshuawuyts showed in the last engines meeting this introduces an undesired coupling that creates a bottleneck in our compilation process.

Reply:
I don't really want to do that refactoring in this PR. Also that would mean the code quality would take a hit, and would make the quality of the code in the connector worse.

Reply:
Doing this in the connector means we cannot use traits, and we must also move all the other code out from here.

Reply:
If @yoshuawuyts thinks this is a good idea to move these out, I'd like him to show us the good way of making it happen without making the code quality drop.

Reply:
I moved them out from the trait due to us really not needing it here.

Reply:
Ok. Just wanted to bring it up. Left a ✅